app/assets/javascripts/components/common/campaign_navbar.jsx

```diff
@@ -1,9 +1,29 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import PropTypes from 'prop-types';
 import { NavLink } from 'react-router-dom';
 import CourseUtils from '../../utils/course_utils';
 
 const CampaignNavbar = ({ campaign }) => {
+  const [activeTab, setActiveTab] = useState(localStorage.getItem('activeTab') || 'overview');
+  const handleTabClick = (tabName) => {
+    setActiveTab(tabName);
+    localStorage.setItem('activeTab', tabName);
+  };
+
+  useEffect(() => {
+    const handleStorageChange = () => {
+      const storedTab = localStorage.getItem('activeTab');
```

Comment:
Hmm... I feel like there must be a better way to determine the active tab than to rely on clicks and localStorage. It should be calculable just from the current path, without using listeners or local strorage.

Reply:
Ok I will look into this.